packages/google_maps_flutter/google_maps_flutter_web/example/integration_test/shapes_test.dart

```diff
@@ -10,6 +10,8 @@ import 'package:integration_test/integration_test.dart';
 import 'package:google_maps_flutter_platform_interface/google_maps_flutter_platform_interface.dart';
 import 'package:google_maps_flutter_web/google_maps_flutter_web.dart';
 import 'package:google_maps/google_maps.dart' as gmaps;
+import 'package:google_maps/google_maps_visualization.dart'
+    as gmaps_visualization;
```

Comment:
`as visualization` is probably good enough (look at the geometry one below)

Reply:
Done